tests/test_merge_refs.py

```diff
@@ -0,0 +1,163 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from virtool_cli.merge_refs import organize_otus_by_taxid, organize_isolates_by_source_name, write_cache, \
+    update_cache, get_cache, compare_isolates, create_otu_results_path, write_isolates, OTUInfo, IsolateInfo
+
+
+@pytest.mark.asyncio
+async def test_organize_otus_by_taxid():
+    source_src_path = Path("./tests/files/source_ref/src")
+    target_src_path = Path("./tests/files/target_ref/src")
+
+    otu_paths_by_taxid, no_taxid_found = await organize_otus_by_taxid(source_src_path, target_src_path)
+
+    assert otu_paths_by_taxid == {"665102": [OTUInfo(ref="source", name="abutilon_brazil_virus"),
+                                             OTUInfo(ref="source", name="abutilon_brazil_virus")],
+                                  "438782": [OTUInfo(ref="source", name="abaca_bunchy_top_virus"),
+                                             OTUInfo(ref="source", name="abaca_bunchy_top_virus")],
+                                  "190811": [OTUInfo(ref="source", name="bamboo_mosaic_virus_satellite_rna"),
+                                             OTUInfo(ref="source", name="bamboo_mosaic_virus_satellite_rna")],
+                                  "35286": [OTUInfo(ref="source", name="bamboo_mosaic_virus"),
+                                            OTUInfo(ref="source", name="bamboo_mosaic_virus")],
+                                  "2060511": [OTUInfo(ref="source", name="babaco_mosaic_virus"),
+                                              OTUInfo(ref="source", name="babaco_mosaic_virus")]}
+
+    assert no_taxid_found == [
+        {"_id": "foobar",
+         "name": "test",
+         "abbreviation": "",
+         "schema": [{"name": "foo",
+                     "molecule": "bar",
+                     "required": True}],
+         "taxid": None
+         }]
+
+
+@pytest.mark.asyncio
+async def test_organize_isolates_by_source_name():
+    otu_infos = [OTUInfo(ref="source", name="abutilon_brazil_virus"),
+                 OTUInfo(ref="target", name="abutilon_brazil_virus")]
+
+    result = await organize_isolates_by_source_name(
+        otu_infos,
+        "./tests/files/source_ref/src",
+        "./tests/files/target_ref/src"
+    )
+
+    expected = {"unknown": [IsolateInfo(ref="source", otu_name="abutilon_brazil_virus", id="un404hxp"),
+                            IsolateInfo(ref="target", otu_name="abutilon_brazil_virus", id="un404hxp")]}
```

Comment:
This is how this should look to be readable. Please update other similar blocks accordingly.
```suggestion
expected = {
    "unknown": [
        IsolateInfo(ref="source", otu_name="abutilon_brazil_virus", id="un404hxp"),
        IsolateInfo(ref="target", otu_name="abutilon_brazil_virus", id="un404hxp")
    ]
}
```